npa.js

```diff
@@ -2,7 +2,6 @@ var url = require("url")
 var assert = require("assert")
 var util = require("util")
 var semver = require("semver")
-var path = require("path")
```

Comment:
To avoid confusion this should be in a separate commit (or PR; I think it may be time to `standard`ize this code base).